waspc/src/Wasp/JsImport.hs

```diff
@@ -2,23 +2,21 @@ module Wasp.JsImport
     ( JsImport(..)
     ) where
 
-import Data.Aeson ((.=), object, ToJSON(..))
-
-import StrongPath (Path, Rel, File)
-import qualified StrongPath as SP
-import ExternalCode (SourceExternalCodeDir)
+import           Data.Aeson (ToJSON (..), object, (.=))
+import qualified Path.Posix as PPosix
 
 
 -- | Represents javascript import -> "import <what> from <from>".
 data JsImport = JsImport
     { _defaultImport :: !(Maybe String)
-    , _namedImports :: ![String]
-    , _from :: !(Path (Rel SourceExternalCodeDir) File)
+    , _namedImports  :: ![String]
+    -- Relative to source external code dir.
+    , _from          :: !(PPosix.Path PPosix.Rel PPosix.File)
```

Comment:
Unfortunately, here we lost StrongPath. The thing was, this has to be posix path, it can't depend on current OS that wasp is running on, it always has to be the same, always `/`, because it is how paths in JS imports are defined.

Could I incorporate StrongPath here? Maybe I am not sure yet hm, I will think about that more.